GitUI/FilterRevisionsHelper.cs

```diff
@@ -6,19 +6,19 @@ namespace GitUI
 {
     public class FilterRevisionsHelper : IDisposable
     {
-        private ToolStripTextBox _NO_TRANSLATE_textBox;
-        private ToolStripDropDownButton _NO_TRANSLATE_dropDownButton;
-        private RevisionGrid _NO_TRANSLATE_revisionGrid;
-        private ToolStripLabel _NO_TRANSLATE_label;
-        private ToolStripButton _NO_TRANSLATE_showFirstParentButton;
+        private ToolStripTextBox _nO_TRANSLATE_textBox;
```

Comment:
❗️ `_NO_TRANSLATE_` appears to be a prefix with special meaning.

Reply:
Yes, it is.
Can we add into some sort of a dictionary?

Reply:
Most `_NO_TRANSLATE_*` fields are defined in `*.Designer.cs` files which SCA leaves alone. Both `FilterBranchHelper` and `FilterRevisionsHelper` are non-partial classes.

I don't know of a dictionary. The Hungarian prefix config only allows 1 or 2 characters in lowercase. Perhaps @sharwell has a suggestion.

All that aside, if I capitalise the `N` character, the analyzer still accepts them, so I have renamed them back to their former glory in bc41dc8ac. Though I feel like these names shouldn't have passed the analyzer somehow...